1.0/citeme.bib

```diff
@@ -0,0 +1,7 @@
+@techreport{perry2012ogc,
+  title={OGC GeoSPARQL-A geographic query language for RDF data},
```

Comment:
please just format as per 1.1 with spaces around the '-'.

Reply:
Done